modules/navier_stokes/src/fvkernels/WCNSFV2PMomentumAdvectionSlip.C

```diff
@@ -27,6 +27,9 @@ WCNSFV2PMomentumAdvectionSlip::validParams()
   params.addParam<MooseFunctorName>("w_slip", "The slip velocity in the z direction.");
   params.addRequiredParam<MooseFunctorName>("rho_d", "Dispersed phase density.");
   params.addParam<MooseFunctorName>("fd", 0.0, "Fraction dispersed phase.");
+  params.renameParam("fd", "fraction_dispersed", "");
+  params.setDocString(NS::density, "Main phase (not dispersed) density functor");
```

Comment:
Might want to rethink this doc string based off what we decide for the phase names

Reply:
kernel syntax is in a tough spot for naming for sure with all the use cases possible. I might leave this
main & dispersed are not shocking